docs/examples/plot_12_async.py

```diff
@@ -0,0 +1,77 @@
+"""
+# Support for asynchronous code
```

Comment:
Very nice example @pmeier ! I have two comments on the text :

 - The text could maybe be improved in order to be more oriented towards the feature that is now available, and not towards the path that led to this feature being available. I'll try to suggest some modifications in this sense.
 - if all of this is automatic without particular configuration we should maybe say it explicitly. Again, I'll try to suggest something.